inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -655,6 +655,8 @@ private function add_used_fonts_preload( string $html, string $used_css ): strin
 
 			$font_url = $this->extract_first_font( $font_face['content'] );
 
+			$font_url = apply_filters( 'rocket_font_url', $font_url );
```

Comment:
Can you add inline doc for the filter, following this format: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters